packages/carbon-graphs/src/js/controls/Graph/helpers/helpers.js

```diff
@@ -925,7 +927,28 @@ const drawNoDataView = (config, svg) => {
   return svg;
 };
 
+/**
+ * Translate a SVG element so it's within the canvas
+ *
+ * @private
+ * @param {d3.selection} canvasSVG - d3 selection node of canvas svg
+ * @param {object} style - element to translate
+ * @param {object} config - config object derived from input JSON
+ * @returns {undefined}
+ */
+const translateSVGElement = (canvasSVG, style, config) => {
```

Comment:
since we added a new function, we should have some unit tests for it.

Reply:
After discussion, we concluded that it's not possible to unit test this function due to its design as part of Carbon's architecture.